tests/utils/editor_recommendation_test.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+
+from data_science_pipeline.utils.editor_recommendation import (
+    get_author_ids_of_given_version_of_paper
+)
+
+MANUSCRIPT_VERSION_1 = "match_version_id_1"
+
+AUTHOR_ID_LIST_1 = [
+    "match_author_id1",
+    "match_author_id2"
+]
+
+MANUSCRIPT_VERSION_2 = "unmatch_version_id_1"
+
+AUTHOR_ID_LIST_2 = [
+    "unmatch_author_id1"
+]
+
+MANUSCRIPT_DICT = {
+  "version_id": {
```

Comment:
just in case it is useful, you could also create the DataFrame from a list of dictionaries, which might make it easier to see what data belongs together. e.g.:

```python
pd.DataFrame([{
    'version_id': MANUSCRIPT_VERSION_1,
    'author_person_ids': AUTHOR_ID_LIST_1
}, {
    'version_id': MANUSCRIPT_VERSION_2,
    'author_person_ids': AUTHOR_ID_LIST_2
}])
```